src/coinqvest/languages/coinqvest.pot

```diff
@@ -451,6 +451,10 @@ msgstr ""
 msgid "Mobile number"
 msgstr ""
 
+#: frontend/class-checkout-form.php:206
+msgid "Phone Number"
```

Comment:
@anexgo Not sure if this is the right way to add translation tokens. It looks auto generated... can you check please and instruct?

Reply:
Yes, it is auto generated. We just need to re-generate the file from code source in Poedit.